docs: refactor-run-a-node (#3244)
OffchainLabs/arbitrum-docs: docs/run-arbitrum-node/02-run-full-node.mdx

content_type: how-to
---

import Tabs from '@theme/Tabs';
import TabItem from '@theme/TabItem';
import DAOChainsExample from './partials/run-full-node/_dao-chains-example.mdx';
import OrbitChainsExample from './partials/run-full-node/_orbit-chains-example.mdx';

:::info Prerequisites

> This page assumes you have completed the steps from the [Start here](/run-arbitrum-node/start-here.mdx) page; if you have not you will need to do so as it gathers RPC endpoints, Nitro version, database snapshots, etc.) that is required to run the node.

Reviewer: ```suggestion
This page assumes you've completed the steps from the [Start here](/run-arbitrum-node/start-here.mdx) page. If you haven't, you'll need to do so, as it gathers RPC endpoints, Nitro version, database snapshots, and other information required to run the node.
```
